src/deploydocs.jl

```diff
@@ -480,7 +480,7 @@ Disassemble repo address into user, host, and path to repo. If no user is given,
 """
 function user_host_upstream(repo)
     # If the repo path contains the protocol, throw immediately an error.
-    occursin(r"^[a-z]+://", repo) && error("The repo path $(repo) should not contain the protocol")
+    occursin(r"^[a-z]+://", repo) && error("The repo path $(repo) should not contain the protocol 'https://'")
```

Comment:
Nitpick, but the regex matches _any_ protocol:

```suggestion
    occursin(r"^[a-z]+://", repo) && error("The repo path $(repo) should not contain the protocol (e.g. 'https://')")
```